tools/terrain/NREL.yaml

```diff
@@ -0,0 +1,29 @@
+solver: "amrWind"
```

Comment:
We should remove these python scripts. There are no tests for these and we don't support python in the CI so these are liable to drift. We need to think of a better place to put these things. Right now it feels like they would belong to the amr-wind-frontend. Any thoughts on where this could live?

Reply:
I am not sure of the best way. I can create a separate toolkit GitHub repo for terrain. 

The setting up of a terrain case by hand is not feasible and this was the reason I had to create the python codes. The user typically creates the terrain and then zooms in to see that the region of interest is sufficient for him and again reruns the script to create the workflow. Also when we create the refinement boxes, the python script lets us keep it at the terrain height and reduces the marking of unnecessary cells.

Reply:
This sure sounds useful. I would recommend a bunch of hardening: linting, proper packaging (with clear dependencies, python versions, etc), scripting (users don't edit the scripts, they pass arguments to the scripts).

The amr-wind-frontend could be a good place since that's what a lot of people are using to set up complex cases. I would hesitate to create yet another repo...

Reply:
@hgopalan can you comment on the status of this review point?

Reply:
We have moved the python code to mmctools and will not be included here.